### DISCUSSION TITLE
docker secrets and scrolling preview

### DISCUSSION THREAD

**Discussion by Massa-sapien** [jordan-dalby/ByteStash#94] [Ideas]
add support for docker secrets ( _FILE doesn't work )
and scrolling preview in the main window of these few lines
Really great work, great app for a developer, Thanks for your work!

**Comment by jordan-dalby** [comment]
@Massa-sapien 
Hi, thanks!
Docker secrets will be in the next release. About scrolling previews- I don't think this will be added since the whole idea is to just show a preview, you can show more preview lines in the settings if you wish.

**Reply by Massa-sapien** [reply]
I was more interested in the ability to scroll (without up, down or left, right bars) along with the ability to select a visible code fragment and copy it. Such a solution would be more useful for short notes, at least for me. I use a similar application from Pawel Malak and such functionality would be more convenient than opening several views of each one separately. But ok. Thanks for your interest.

**Reply by jordan-dalby** [reply]
> I was more interested in the ability to scroll (without up, down or left, right bars) along with the ability to select a visible code fragment and copy it. Such a solution would be more useful for short notes, at least for me. I use a similar application from Pawel Malak and such functionality would be more convenient than opening several views of each one separately. But ok. Thanks for your interest.

Can you explain this more? I don't think I understand what you're asking for. Specifically "opening several views of one separately"

**Reply by Massa-sapien** [reply]
You could then copy directly from the main screen, without opening every window.
Believe me, the first thing I did after installing your version of the application was to reflexively start spinning the mouse wheel on a single view, and try to select those few visible lines.
Scrolling would be useful, but without the bars that would shrink the window.

**Reply by Massa-sapien** [reply]
The view of the code lines in these windows is what distinguishes your application from others. You could develop this functionality, and not create another code editor, that's what code editors are for, I don't understand people who ask for the functionality that I will make the next code editor out of this application. Remember that simplicity is desirable here, and in the flood of functions it is easy to get lost and discouraged.

**Reply by jordan-dalby** [reply]
@Massa-sapien there is a copy button on the preview window though, can't you just use that to copy?

**Reply by Massa-sapien** [reply]
but I just want to copy a few lines. it's just an idea, and you can do whatever you want. Thanks.
